includes/functions.php

```diff
@@ -191,6 +191,19 @@ function convertkit_get_settings_link( $query_args = array() ) {
 
 }
 
+/**
+ * Helper method to return the URL the user needs to visit to register a ConvertKit account.
+ *
+ * @since   1.9.8.5
+ *
+ * @return  string  ConvertKit Registration URL.
+ */
+function convertkit_get_registration_url() {
+
+	return 'https://app.convertkit.com/users/signup?utm_source=wordpress&utm_content=convertkit';
```

Comment:
Do we already have a place to keep constants like this? It might make sense to keep that there then get rid of this function.